src/css/structure/modules/infragistics.ui.tree.css

```diff
@@ -125,6 +125,21 @@
 	height:1.6em;
 }
 
+/* Marin Popov, 26.06.2017, Bug 236973 - Checkboxes cannot be clicked */
+.ui-igtree  .ui-igcheckbox-normal-on {
+	position:relative;
+	top:0;
+	left:0;
+	margin-top:0;
+	margin-left:0;
+}
+.ui-igtree .ui-igcheckbox-normal-off,
+.ui-igtree .ui-igcheckbox-normal-on {
+	width:100%!important;
```

Comment:
@kdinev I need this because of another important rule which is located in the Infragistics theme

/* icon SIZING FIXES for controls which expect a specific icon size */
.ui-iggrid tr .ui-icon,
.ui-igtree .ui-icon,
.ui-igedit .ui-icon,
.ui-igsplitter .ui-icon,
.ui-igcombo .ui-icon {
  height: 16px !important;
  width: 16px !important;
  font-size: 11px;
}

I can remove the row with:.ui-igtree .ui-icon, but I don't know if that will not break anything else